### ISSUE TITLE
KW18 training

### ISSUE THREAD

**Issue by changcheng967** [changcheng967/Kata_web#25]
#### **Backlog**

**Item:** Change training parameters and commands for KW18
**Assignee:** changcheng967
**Priority:** P0
**Description:**
Update and execute KW18 training run to surpass baseline Elo and improve stability. Adjust batch size, samples per epoch, and learning rate. Single A4000 GPU, FP16 enabled.

**Training Command:**

```bash
python train.py \
  -traindir /notebooks/katago/shuffledata \
  -datadir /notebooks/katago/shuffledata \
  -exportdir /notebooks/katago/models \
  -exportprefix "KW18-b28c512nbt-0929" \
  -initial-checkpoint /notebooks/katago/models/KW17-b28c512nbt-0928-s10907327168-d700707/model.ckpt \
  -use-fp16 \
  -batch-size 80 \
  -pos-len 19 \
  -max-epochs-this-instance 1 \
  -samples-per-epoch 200000 \
  -lr-scale 0.2 \
  -epochs-per-export 1 \
  -gnorm-clip-scale 0.8 \
  -sub-epochs 1
```

**Subtasks / Checklist:**

* [x] Copy KW17 checkpoint to KW18 folder
* [x] Update training parameters
* [x] Run first KW18 epoch
* [x] Monitor training loss and Weighted Elo
* [x] Export model after epoch
* [x] Evaluate against KW17 and baseline
* [x] Adjust parameters if instability occurs

---

#### **Ready**

**Item:** Ready to start KW18 epoch
**Description:**
All setup completed, checkpoint copied, parameters updated. Training can be started.
**Subtasks:**

* [x] Confirm data directory contains correct self-play games
* [x] Confirm GPU availability (A4000)
* [x] Verify batch size and samples-per-epoch settings

---

#### **In Progress**

**Item:** KW18 Epoch Running
**Description:**
The KW18 training epoch is running on A4000 GPU. Monitor stability and progress.
**Subtasks:**

* [x] Track training loss curves
* [x] Track Weighted Elo after each export
* [x] Record any warnings/errors

---

#### **In Review**

**Item:** Evaluate KW18 after Epoch
**Description:**
Evaluate KW18 model against KW17 and baseline to confirm improvement.
**Subtasks:**

* [x] Run evaluation games (20–50)
* [x] Record win rate, Elo difference, Weighted Elo
* [x] Document evaluation results on GitHub board

---

#### **Done**

**Item:** KW18 Epoch Completed
**Description:**
KW18 epoch fully trained, exported, and evaluated. Ready for next iteration.
**Subtasks:**

* [x] Model exported to `/notebooks/katago/models/KW18-b28c512nbt-0929`
* [x] Metrics logged: Win rate, Elo, Weighted Elo
* [x] Notes for next KW19 iteration

---

**Comment by changcheng967** [comment]
Removed old training data, downloading newest(2025-09-26)

**Comment by changcheng967** [comment]
Extracted, starting shuffle

**Comment by changcheng967** [comment]
First shuffle attempt, failed starting second attempt(may take a long time)

**Comment by changcheng967** [comment]
The training doesn't look very great

```
📊 Training Log Detailed Analysis
- Entries analyzed: 4

Entry 01 | Samples=2866944 | pacc1=0.5281 | loss=49.5799 | p0loss=1.6145 | p1loss=0.2834
Entry 02 | Samples=2874944 | pacc1=0.5263 | loss=49.6212 | p0loss=1.6241 | p1loss=0.2841 Δpacc1=-0.0018 (-0.33%), Δloss=+0.0414 (+0.08%)
Entry 03 | Samples=2882944 | pacc1=0.5249 | loss=49.6340 | p0loss=1.6303 | p1loss=0.2843 Δpacc1=-0.0014 (-0.27%), Δloss=+0.0127 (+0.03%)
Entry 04 | Samples=2890944 | pacc1=0.5229 | loss=49.6633 | p0loss=1.6387 | p1loss=0.2850 Δpacc1=-0.0020 (-0.38%), Δloss=+0.0294 (+0.06%)
```

**Comment by changcheng967** [comment]
Restarted training, monitoring training log
